dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -2318,7 +2266,49 @@ private void deletePermission(Permission p) {
 
 	}
 
-	private List<Permission> filterOnlyNonInheritablePermissions(List<Permission> permissions, String permissionableId) {
+    @WrapInTransaction
+	private void deleteInsertPermission(Permissionable permissionable, String type,
+            Permissionable newReference) throws DotDataException {
+
+        final String permissionId = permissionable.getPermissionId();
+
+        try{
+            Logger.debug(this.getClass(), "PERMDEBUG: " + Thread.currentThread().getName() + " - " + permissionId
+                    + " - started");
+
+            DotConnect dc1 = new DotConnect();
+            dc1.setSQL("SELECT inode FROM inode WHERE inode = ?");
+            dc1.addParam(permissionId);
+            List<Map<String, Object>> l = dc1.loadObjectResults();
```

Comment:
rename l to inodeList
and the other one to identifierList

Reply:
done